src/Microsoft.VisualBasic.Core/tests/FileSystemTests.cs

```diff
@@ -723,7 +723,7 @@ public void Write_ArgumentException()
             {
                 ex = e;
             }
-            Assert.NotNull(ex?.GetType() == typeof(TException));
+            Assert.True(ex?.GetType() == typeof(TException));
```

Comment:
I'd recommend just fixing this:

```suggestion
            if (ex == null)
            {
                throw new ThrowsException(typeof(TException));
            }
            
            if (ex.GetType() != typeof(TException))
            {
                throw new ThrowsException(typeof(TException), ex);
            }
```

Which may require a using-import for `Xunit.Sdk`.

https://github.com/dotnet/corefx/blob/30ca4113ed07fc78060a0c9d3f95eee4fe8f8ee3/src/System.Text.Json/tests/JsonTestHelper.cs#L698-L724